tests/collector.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
```

Comment:
Yeah, but I noticed that you have added a single entry point for all tests in one of the commits in #87 yet in this PR and the other one a test file is added but it's just dead code that isn't being run, so I assumed that you're just waiting for that PR to land to add these tests to that entry point.